x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_event_log_list_status_filter.tsx

```diff
@@ -8,10 +8,9 @@
 import React, { useState, useCallback } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { EuiFilterButton, EuiPopover, EuiFilterGroup, EuiFilterSelectItem } from '@elastic/eui';
-import type { EcsEventOutcome } from '@kbn/core/server';
-import { RuleEventLogListStatus } from './rule_event_log_list_status';
+import { KibanaAlertingOutcome, RuleEventLogListStatus } from './rule_event_log_list_status';
 
-const statusFilters: EcsEventOutcome[] = ['success', 'failure', 'unknown'];
+const statusFilters: KibanaAlertingOutcome[] = ['success', 'failure', 'warning'];
```

Comment:
just curious, so we're no longer able to sort by `unknown`, but it seems like we still reference the `unknown` status in `KibanaAlertingOutcome`, can event outcomes still be unknown?

Reply:
The number of unknown outcomes are still calculated in kpi aggregation, and I left it in if we ever wanted to display those values in ui. I can remove it though, since it's not currently being used.

Reply:
Resolved in x-pack/plugins/alerting/server/rules_client/tests/get_execution_log.test.ts